configMobile.js

```diff
@@ -0,0 +1,24 @@
+const config = require( './config.js' );
+
+const BASE_URL = process.env.MW_SERVER;
+const tests = [
+	{
+		label: 'Tree (#minerva #mobile)',
+		path: '/wiki/Tree?useskin=minerva&useformat=mobile'
+	},
+	{
+		label: 'Tree (#minerva #mobile #logged-in)',
+		path: '/wiki/Tree?useskin=minerva&useformat=mobile'
+	}
+];
+
+const scenarios = tests.map( ( test ) => {
+	return Object.assign( {}, test, {
+		url: `${BASE_URL}${test.path}`,
+		delay: 1500
```

Comment:
Is this for the shifting notification icon?

Reply:
Yes this and the main menu animation. We'll be fixing Echo shortly, and I think it might be time to add the prefers reduce motion styles in Vector in core so Minerva gets them too.